docs/release.md

```diff
@@ -8,45 +8,53 @@ For now release artifacts are created only for debian package [repository](https
-- stable releases are built for each commit to `stable` branch when `indy-node` dependency is set to *stable* version.
+- `dev` releases are built for each commit to `master` branch;
+- `rc` (release candidate) releases are built for each commit to `stable` when `indy-node` dependency is set to *non stable* version;
+- `stable` releases are built for each commit to `stable` branch when `indy-node` dependency is set to *stable* version.
 
 ## Release Pipeline
 
 1. Preparation
-    1. `indy-node` version is checked.
+    1. `indy-node` dependency version is checked in [sovtoken/setup.py](../sovtoken/setup.py).
     2. Necessary AWS resources are created.
 2. Build
     1. AWS CodeBuild project builds debian packages for each plugin
 3. Publishing and Notification
-    1. Built packages are published to the repository component depending on the `indy-node` dependency version and GitHub branch as it was mentioned above:
-        - `master` commits are published to `master`, debian packages include unique pipeline build number as part of their versions.
-        - `stable` commits are published to `rc` if `indy-node` version is *non stable* (e.g. `1.8.0~rc1`) and to `stable` otherwise.
-          `stable` packages' versions match ones from the source code and `rc` packages will have additional suffix.
+    1. The packages are published to a debian repository channel. The channel depends on `indy-node` dependency version and GitHub branch:
+        - `master` commits are published to `master` debian channel, debian packages include unique pipeline build number as part of their versions (e.g. `1.0.0~dev65`).
+        - `stable` commits are published:
+            - to `rc` channel if `indy-node` version is *non stable* (e.g. `1.8.0~rc1`);
+            - to `stable` channel otherwise.
+        - Note. `stable` packages' versions match ones from the source code (e.g. `0.9.13`) and `rc` packages will have additional suffix (e.g. `0.9.14~rc10`).
 
 ## Release Workflow
 
 1. Release candidate preparation
     1. [**Contributor**]
-        - Creates a branch from `stable`.
-        - Merges necessary changes from `master`;
-        - Waits for `indy-node` release candidate is released and set its versions in [sovtoken/setup.py](../sovtoken/setup.py) and [devops/Makefile](../devops/Makefile).
-        - Creates a pull request to `stable`.
-    2. [**Maintainer**] once CI testing is passed reviews, approves and merges the PR.
-    3. [**build server**]
-        - Once PR is merged starts the [release pipeline](#release-pipeline) which publishes debian packages to `rc` repo component.
+        - Create a branch from `stable`.
+        - Merge necessary changes from `master`;
+        - Wait for `indy-node` release candidate and set its version in [sovtoken/setup.py](../sovtoken/setup.py) and `FPM_P_DEPENDS` variable in  [devops/Makefile](../devops/Makefile).
```

Comment:
It makes sense to mention, that `indy-node`'s version here is RC one with an example.
Also it may make sense to mention, that we may set stable version of indy-node from the very beginning. For example, if we create a hotfix release for plugins without any changes in indy-node.

Reply:
Current release process doesn't supposed such a case. I'm going to fix that and will update the docs then.